internal/lsp/rego/builtins.go

```diff
@@ -7,13 +7,20 @@ import (
 	"github.com/open-policy-agent/opa/ast"
 )
 
-var BuiltInsLock = &sync.RWMutex{}                          // nolint:gochecknoglobals
-var BuiltIns = builtinMap(ast.CapabilitiesForThisVersion()) //nolint:gochecknoglobals
+var builtInsLock = &sync.RWMutex{}                          // nolint:gochecknoglobals
+var builtIns = builtinMap(ast.CapabilitiesForThisVersion()) //nolint:gochecknoglobals
```

Comment:
I think we can drop the nolint comment now.

Reply:
done, forgot what commit it was in though 😅